code/datums/helper_datums/global_iterator.dm

```diff
@@ -109,7 +109,7 @@ Data storage vars:
 			CRASH("The global_iterator loop \ref[src] failed to terminate in designated timeframe. This may be caused by server lagging.")
 	return 1
 
-/datum/global_iterator/proc/process()
+/datum/global_iterator/process()

```

Comment:
Piece of crap global iterators. Space pods are the only reason you exist. WHY CAN'T YOU BE UP TO DATE LIKE YOUR SUPERIOR SENIOR AND PREDECESSOR.